content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -66,13 +66,21 @@ program accepts a file as an argument and prints its content reversed.
 rev data.txt
 ```
 
-For this example, you'll use the `rev` tool from the
+You'll use the `rev` tool from the
```

Comment:
Line 69 seems confusing. Could you combine that sentence with lines 61-63?

Reply:
We could omit this paragraph - @alculquicondor is that OK?

Reply:
SGTM